### PR TITLE
Update after-effects-quiz.md

### PR DESCRIPTION
**Pull request by rajdeepsri**

### CODE REVIEW

File: after-effects/after-effects-quiz.md [hunk 1]
```diff
@@ -74,10 +74,10 @@
 
 #### Q11. What properties are added to a 3D layer? (Select three items.)
 
-- [ ] Z Rotation
-- [ ] Anchor Point
+- [x] Z Rotation
+- [x] Anchor Point
 - [ ] Material Options
-- [ ] Position
+- [x] Position
```
Comment (Ebazhanov) [inline after-effects/after-effects-quiz.md:80]: it is better to keep only one, can you leave your favorite, thanks

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@rajdeepsri left comment

File: after-effects/after-effects-quiz.md (hunk 1 above)
Comment (rajdeepsri) [reply after-effects/after-effects-quiz.md:80]: @Ebazhanov can you please elaborate i didn't understand.

Comment (Ebazhanov) [reply after-effects/after-effects-quiz.md:80]: sure, please leave only one answer with [x]

Comment (rajdeepsri) [reply after-effects/after-effects-quiz.md:80]: @Ebazhanov actually, the question asked to select "three items", that is why I selected three options. Hope you understand this

Comment (Ebazhanov) [reply after-effects/after-effects-quiz.md:80]: my bad did not read it properly, you are right!!!

Comment (rajdeepsri) [reply after-effects/after-effects-quiz.md:80]: yeah it happens, by the way thanks, I am new to contributing and stuff. Got to learn a lot from all this

**Review by Ebazhanov — APPROVED** [review]
Thanks a lot for your contribution! @rajdeepsri great job 💪

**Comment by Ebazhanov** [comment]
@all-contributors please add @rajdeepsri for design, code

**Pull request merged**